bridgev2/portal.go

```diff
@@ -4046,6 +4046,19 @@ func (portal *Portal) UpdateDisappearingSetting(ctx context.Context, setting dat
 	if portal.MXID == "" {
 		return true
 	}
+
+	timerType := event.DisappearingTypeAfterSend
+	if setting.Type == database.DisappearingTypeAfterRead {
+		timerType = event.DisappearingTypeAfterRead
+	} else if setting.Type == database.DisappearingTypeNone {
+		timerType = event.DisappearingTypeNone
+	}
```

Comment:
It's the same type, you don't need to map anything

Reply:
good catch